src/GitHub.VisualStudio.UI/RepositoryOrigin.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace GitHub.VisualStudio.UI
+{
+    public enum RepositoryOrigin
+    {
+        DotCom,
+        Enterprise,
+        Other,
+        NonGitRepository,
+    }
```

Comment:
Move the `Other` value up to the first position, or add a `None` value. enums always have a default value, and distinguishing between `DotCom` and `haven't probed yet / this is a default` is hard if they're the same value.